e2e/etcdctl_test.go

```diff
@@ -60,13 +60,159 @@ func etcdctlSet(epc *etcdProcessCluster, key, value string, noSync bool) error {
 		endpoint = backends[0].cfg.acurl.String()
 	}
 
-	putArgs := []string{"../bin/etcdctl", "--endpoint", endpoint}
+	args := []string{"../bin/etcdctl", "--endpoint", endpoint}
+	if noSync {
+		args = append(args, "--no-sync")
+	}
+	args = append(args, "set", key, value)
+
+	return spawnWithExpect(args, value)
+}
+
+func etcdctlGet(epc *etcdProcessCluster, key, value string, noSync bool) error {
+	endpoint := ""
+	if proxies := epc.proxies(); len(proxies) != 0 {
+		endpoint = proxies[0].cfg.acurl.String()
+	} else if backends := epc.backends(); len(backends) != 0 {
+		endpoint = backends[0].cfg.acurl.String()
+	}
+
+	args := []string{"../bin/etcdctl", "--endpoint", endpoint}
+	if noSync {
+		args = append(args, "--no-sync")
+	}
```

Comment:
this is repeated for every etcdctl function; fold into `etcdctlPrefixArgs` like `cURLPrefixArgs`